src/Microsoft.Android.Build.BaseTasks/Microsoft.Android.Build.BaseTasks.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
     <RootNamespace>Microsoft.Android.Build.Tasks</RootNamespace>
+    <LangVersion>latest</LangVersion>
```

Comment:
I added this, otherwise the `netstandard2.0` target couldn't build `LockFile.cs`.